R/utility-job.R

```diff
@@ -533,9 +523,7 @@ waitForTasksToComplete <-
         jobId)
       }
 
-      if (taskCounts$completed >= totalTasks &&
```

Comment:
Counting service notoriously runs into problems. Might need to change to doing smart list calls in the future.

Reply:
I've added job state in the check